### DISCUSSION TITLE
Text not aligning to logo

### DISCUSSION THREAD

**Discussion by loustact** [fastfetch-cli/fastfetch#1238] [Q&A] [answered]
Just tried out wezterm on Windows since it seems that Windows Terminal doesn't support images and I really want to explore alternatives. I both have wezterm and fastfetch installed through Scoop. It seems when I apply the image logo through iterm, the text misaligns, bein
g a bit higher than usual. Is there any fix for it? Thank you :)
![wezterm-gui_WLn2Q2DnQC](https://github.com/user-attachments/assets/d948babb-13c7-4f81-9415-a16c9cc2e279)

**Comment by CarterLi** [comment]
Your configuration please.

**Reply by loustact** [reply]
Whoops, here's my config [config.jsonc](https://github.com/user-attachments/files/16833038/config.jsonc)

Edit: Also, here's the photo [output.png](https://github.com/user-attachments/assets/cf28e75b-59f5-4410-a504-9c0237eef27e)

**Reply by loustact** [reply]
Is my wezterm config also needed?

**Comment by CarterLi** [comment]
`clear; fastfetch --logo-padding-top 1` works for me

**Reply by CarterLi** [reply, marked as answer]
Image logo support on Windows is known to be very buggy. See https://github.com/wez/wezterm/issues/4240

**Reply by loustact** [reply]
Oof, I'll take note of that. Thank you :)
